pages/linux/sherlock.md

```diff
@@ -0,0 +1,32 @@
+# sherlock
+
+> Find usernames across social networks.
+
+- Display help:
+
+`sherlock --help`
+
+- Save result to file (if using single username):
+
+`sherlock {{username}} --output {{filename}}`
+
+- Save result to folder (if using multiple usernames):
+
+`sherlock {{[usernames]}} --folderoutput {{foldername}}`
+
+- Make requests over Tor:
+
+`sherlock --tor {{username}}`
+
+- Make requests over Tor with new Tor circuit after each request:
+
+`sherlock --unique-tor {{username}}`
+
+- Make requests over a proxy:
+
+`sherlock {{username}} --proxy {{proxyurl}}`
+
+- Browse to all results on default browser:
```

Comment:
This example open a report? I don't get it.

Reply:
It finds the username in the social networks and opens the link of the profiles.

Reply:
Perhaps is better
```suggestion
- Search for a specific username on social networks and open results in the default web browser:
```